app/src/main/java/at/roteskreuz/stopcorona/model/repositories/ReportingRepository.kt

```diff
@@ -167,10 +249,10 @@ class ReportingRepositoryImpl(
 
     private suspend fun uploadData(
         warningType: WarningType,
-        temporaryExposureKeys: List<TemporaryExposureKey>
+        temporaryExposureKeys: List<Pair<List<TemporaryExposureKey>, UUID>>
```

Comment:
why not use the TemporaryExposureKeysWrapper?

Reply:
Because the TemporaryExposureKeysWrapper is used in database operations where the TemporaryExposureKey it's not necessary (only the `rollingStartIntevalNumber`) and would be an overhead to retrieve it.

For the updating of our state (y->r, r->g, y-> g) there are situations where we need to update the state of the temporary exposure keys that are already in db, to perform this operation only the rollingStartInterval is needed and it would be an overhead to search for the associated keys.

Reply:
It could be some named data class, but let's keep it.